src/main/webapp/course-style.css

```diff
@@ -0,0 +1,78 @@
+.flex-container {
+    display: flex;
+    height: 600px;
+}
+
+.flex-child {
+    flex: 1;
+}  
+
+.form-container {
+    padding-top: 100px;
+}
+
+.my-3 {
+    margin: 20px 125px;
+}
+
+input[type="number"],
+textarea {
+  border: 1px solid #228b22;
+  border-radius: 4px;
+  box-sizing: border-box;
```

Comment:
border-box is often something you want for either everything or nothing so might be worth putting at the top level. 

Often people like to do something like
```
/* Remove padding and margin */ 
*  
    { 
    margin: 0; 
    padding: 0; 
    border: 0; 
}
```

In some index.css that gets added to every page (* means it hits all elements). 
Adding box-sizing to something like this could be good.

Btw * as a selector isn't performant technically but it hardly matters for your use case.